x/evm/statedb/journal.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2016 The go-ethereum Authors
+// This file is part of the go-ethereum library.
```

Comment:
would be nice to add a test file and add unit tests

Reply:
https://github.com/tharsis/ethermint/issues/876